query-primitives/src/main/scala/io/shiftleft/queryprimitives/utils/ExpandTo.scala

```diff
@@ -122,20 +105,29 @@ object ExpandTo {
     }
   }
 
-  def methodToOutParameters(method: Vertex): Seq[Vertex] = {
+  def methodToOutParameters(method: Vertex): Seq[Vertex] =
     method
       .vertices(Direction.OUT, EdgeTypes.AST)
       .asScala
       .filter(_.isInstanceOf[nodes.MethodParameterOut])
       .toSeq
-  }
 
-  def allCfgNodesOfMethod(method: Vertex): TraversableOnce[Vertex] = {
+  def allCfgNodesOfMethod(method: Vertex): TraversableOnce[Vertex] =
     method.vertices(Direction.OUT, EdgeTypes.CONTAINS).asScala
-  }
 
-  def reference(node: Vertex): Option[Vertex] = {
+  def reference(node: Vertex): Option[Vertex] =
     node.vertices(Direction.OUT, EdgeTypes.REF).nextOption
-  }
+
+  /**
+    * For a given name, determine whether it is the
+    * name of a "member access" operation, e.g.,
+    * "<operator>.memberAccess".
+    * */
+  def isGenericMemberAccessName(name: String): Boolean =
```

Comment:
That appears like a weird choice for the location of this method.
It is only vaguely related to the other methods which perform the actual expansions.
I'm not saying that `queryprimitives` was better, but at least it didn't have any meaning attached to it.